cms/templates/index.html

```diff
@@ -108,7 +108,7 @@ <h3 class="title">${_("Create a New Course")}</h3>
                   <label for="new-course-run">${_("Course Run")}</label>
                   ## Translators: This is an example for the "run" used to identify different
                   ## instances of a course, seen when filling out the form to create a new course.
-                  <input class="new-course-run" id="new-course-run" type="text" name="new-course-run" required placeholder="${_('e.g. 2014_T1')}" aria-describedby="tip-new-course-run tip-error-new-course-run" />
+                  <input onkeyup="textDirection(this);" class="new-course-run input-placeholder" id="new-course-run" type="text" name="new-course-run" required placeholder="${_('e.g. 2014_T1')}" aria-describedby="tip-new-course-run tip-error-new-course-run" />
```

Comment:
@asadiqbal08 -- I'm not seeing how the onkeyup attribute ties into this -- are you suggesting that we use this event as the LTR/RTL switch?